src/web/health_handler.go

```diff
@@ -0,0 +1,26 @@
+package web
+
+import (
+	"time"
+	"github.com/gin-gonic/gin"
+	"github.com/siesgstarena/epicentre/src/services/logger"
+)
+
+type health struct {
+    Name     		string `json:"name"`
+	Description     string `json:"description"`
+	Health 			string `json:"health"`
+	Timestamp       time.Time `json:"timestamp"`
+}
+
+// HeathHandler Sends info about health of API
+func HeathHandler(c *gin.Context)  {
+	logger.Log.Info("Inside Health Handler")
```

Comment:
don't have to log inside handlers unless custom or complex operations.. we will use gin to log http requests.

Reply:
Sure. Gin is logging our every request just like morgan